src/components/UserSettings/style.css

```
@@ -1,3 +1,12 @@
 .half-width {
     width: 50%;
+}
+
+:global(.pf-c-nav__list.card-pf) {
```

```suggestion
.nav-list-container {
```

And then in `GlobalSettings` do 
```jsx
  <NavList className={style['nav-list-container']}>
```

---

We need to override `card-pf` but it's definitely better to narrow the scope as much as possible. What do you think about adding a marker class just for this purpose? Take a look at newest version.

---

Yeah the extra marker class looks good.